docs/getting-started/auth-self-defence-checks.adoc

```diff
@@ -0,0 +1,85 @@
+= Self Defence Checks
+
+As part of the Auth SDK module, there is a self defence check feature.  This feature offers multiple checks that can be used to analyse the secuirty of the device.
+
+== Prequisite's
+
+. Import and initialise the Auth SDK module - link:auth.adoc[AeroGear Services Auth SDK].
+
+== Usage
+
+The `SecurityService` can be retrieved using the `MobileCore#getInstance` method.
+
+[source, java]
+----
+SecurityService securityService = mobileCore.getInstance(SecurityService.class);
+----
+
+Any subsequent `MobileCore#getInstance` method will return the same instance of `SecurityService`.
+
+The `SecurityService#check` method can be used to run different secuirty checks.  It expects a `SecurityCheckType` paramater and returns `SecurityCheckResult`.
+
+[source, java]
+----
+SecurityCheckResult result = securityService.check(SecurityCheckType.<check_type>);
+----
+
+=== Detecting Root Access
+
+To detect if a device has been compromised (rooted/jailbreak) the `SecurityCheckType#IS_ROOTED` can be used.  This function uses the link:https://github.com/scottyab/rootbeer[Rootbeer] library to check if root access is present on the device.
```

Comment:
O wouldn't say that a rooted device is 'compromised'

Reply:
Ya ok, I'll remove that and rephrase. Thanks